pydata_sphinx_theme/static/css/theme.css

```diff
@@ -66,6 +66,7 @@
   --pst-color-sidebar-link: 77, 77, 77;
   --pst-color-sidebar-link-hover: var(--pst-color-active-navigation);
   --pst-color-sidebar-link-active: var(--pst-color-active-navigation);
+  --pst-color-sidebar-caption: 77, 77, 77;
```

Comment:
Maybe re: uppercase, slam that into a variable as well?

```css
--pst-sidebar-caption-transform: uppercase;
--pst-sidebar-font-size: 0.9em;
```

Reply:
@bollwyvl sorry I didn't yet answer / address this comment. I am certainly fine with adding more styling knobs here, but maybe we can wait until someone needs it?